spec/ParseGraphQLServer.spec.js

```diff
@@ -126,8 +126,10 @@ describe('ParseGraphQLServer', () => {
 
     it("should return schema and context with req's info, config and auth", async () => {
       const options = await parseGraphQLServer._getGraphQLOptions();
-      expect(options.multipart).toEqual({
-        fileSize: 20971520,
+      expect(new options.fetchApi.Body().options).toEqual({
```

Comment:
This is an internal property of our implementation. It is not part of Fetch spec. I'm not sure if you should test that.

Reply:
`@whatwg-node/fetch` implements Fetch API right? But Fetch API doesn't have `Body.options`. So that means it is an internal field which is NOT intended to be used by user. 

>  And in case of a fail, it's just a test we will be able to update it easily :)

After the long process in my previous PRs, I don't think it should be that easy :) So I'd not recommend to keep this test like that.